File: 백준/Bronze/25304. 영수증/영수증.py
total = int(input())
n = int(input())
c=0
d=0
for i in range(0,n):
    a, b = map(int,input().split())
    c=int(a)*int(b)
    d += c

if(total==d):
    print("Yes")
else : print("No")

File: 백준/Bronze/25304. 영수증/영수증.kt
import java.util.*

fun main() = with(Scanner(System.`in`)) {
    var total = nextInt()
    var n = nextInt()
    var result : Int = 0

    for(i : Int in 1..n){
        var a = nextInt()
        var b = nextInt()

        result += a*b
    }

    if(total == result){
        print("Yes")
    }else {
        print("No")
    }
}